docs/source/user_guide/runtime-conf.md

```diff
@@ -181,16 +183,23 @@ The namespace used to run your pipeline in Kubeflow Pipelines. This setting is r
 
 Example: `anonymous`
 
+##### Kubeflow authentication type (auth_type)
+Authentication type Elyra uses to gain access to Kubeflow Pipelines. This setting is required. Supported types are:
+- No authentication (`NO_AUTHENTICATION`).
+- Kubernetes service account token (`KUBERNETES_SERVICEACCOUNT_TOKEN`). This authentication type is only supported if Elyra runs as a pod in Kubernetes, e.g. as a Kubeflow notebook server. You must configure a service account token in Kubernetes, as outlined [here](https://www.kubeflow.org/docs/components/pipelines/sdk/connect-api/#multi-user-mode).
+- DEX configured for static password authentication (`DEX_STATIC_PASSWORDS`). This authentication requires a username and a password.
+- DEX configured for LDAP authentication (`DEX_LDAP`). This authentication requires a username and a  password.
+- DEX (`DEX_LEGACY`). Use this type only if none of the other authentication types applies or if your Kubeflow deployment is not configured for any other listed type. This authentication requires a username and a password.
```

Comment:
I think we should state that `DEX_LEGACY` is the same code as Elyra `3.1.1` and earlier.

(Also, that `DEX_LEGACY` is deprecated, and will be removed in a future release)

Reply:
While I agree that it would be desirable, I'm not sure removal will be possible unless we can make the other (auth specific) providers work for everyone (See @portellaa's problem).

Reply:
What kind of auth are they using, do you know how their cluster is set up? 

In my mind, the idea is to only have single-kind auth-types in the future, that will allow us to do advanced error handling and provide useful messages in logs.